roles/variables/vars/main.yml

```diff
@@ -89,6 +89,175 @@ zookeeper_combined_properties: "{{ zookeeper_properties | confluent.platform.com
+      ssl.keystore.location: "{{kafka_controller_keystore_path}}"
+      ssl.keystore.password: "{{kafka_controller_keystore_storepass}}"
+      ssl.key.password: "{{kafka_controller_keystore_storepass}}"
+  fips:
+    enabled: "{{fips_enabled}}"
+    properties:
+      enable.fips: 'true'
+      security.providers: io.confluent.kafka.security.fips.provider.BcFipsProviderCreator,io.confluent.kafka.security.fips.provider.BcFipsJsseProviderCreator
+      ssl.keymanager.algorithm: PKIX
+      ssl.trustmanager.algorithm: PKIX
+      ssl.keystore.type: BCFKS
+      ssl.truststore.type: BCFKS
+      ssl.truststore.location: "{{kafka_controller_truststore_path}}"
+      ssl.truststore.password: "{{kafka_controller_truststore_storepass}}"
+      ssl.keystore.location: "{{kafka_controller_keystore_path}}"
+      ssl.keystore.password: "{{kafka_controller_keystore_storepass}}"
+      ssl.key.password: "{{kafka_controller_keystore_storepass}}"
+      ssl.enabled.protocols: TLSv1.2
+  sasl_enabled:
+    enabled: "{{ kafka_controller_sasl_enabled_mechanisms|length > 0 }}"
+    properties:
+      sasl.enabled.mechanisms: "{% for mechanism in kafka_controller_sasl_enabled_mechanisms %}{% if loop.index > 1%},{% endif %}{{ mechanism|upper }}{% endfor %}"
+  sasl_gssapi:
+    enabled: "{{ 'GSSAPI' in kafka_controller_sasl_enabled_mechanisms }}"
+    properties:
+      sasl.kerberos.service.name: "{{kerberos_kafka_controller_primary}}"
+  sr:
+    enabled: "{{ kafka_controller_schema_validation_enabled and 'schema_registry' in groups }}"
+    properties:
+      confluent.schema.registry.url: "{{schema_registry_url}}"
+  sr_ssl_fips:
+    enabled: "{{ kafka_controller_schema_validation_enabled and 'schema_registry' in groups and schema_registry_ssl_enabled and fips_enabled }}"
+    properties:
+      confluent.ssl.keymanager.algorithm: PKIX
+      confluent.ssl.trustmanager.algorithm: PKIX
+      confluent.ssl.keystore.type: BCFKS
+      confluent.ssl.truststore.type: BCFKS
+  sr_rbac:
+    enabled: "{{ kafka_controller_schema_validation_enabled and 'schema_registry' in groups and rbac_enabled }}"
+    properties:
+      confluent.basic.auth.credentials.source: USER_INFO
+      confluent.basic.auth.user.info: "{{schema_registry_ldap_user | default('sr') }}:{{schema_registry_ldap_password | default('pass')}}"
+  sr_basic:
+    # Should not turn on basic auth if rbac is enabled
+    enabled: "{{ kafka_controller_schema_validation_enabled and 'schema_registry' in groups and schema_registry_authentication_type == 'basic'}}"
+    properties:
+      confluent.basic.auth.credentials.source: USER_INFO
+      confluent.basic.auth.user.info: "{{schema_registry_basic_users_final.admin.principal}}:{{schema_registry_basic_users_final.admin.password}}"
+  rbac:
+    enabled: "{{ rbac_enabled }}"
+    properties:
+      authorizer.class.name: io.confluent.kafka.security.authorizer.ConfluentServerAuthorizer
+      confluent.authorizer.access.rule.providers: CONFLUENT,ZK_ACL
```

Comment:
Let's put a comment here. ZK_ACL will not work with kraft (because, of course, it uses ZK) and add the following link.
https://cwiki.apache.org/confluence/display/KAFKA/KIP-801%3A+Implement+an+Authorizer+that+stores+metadata+in+__cluster_metadata